components/accordion/package.json

```diff
@@ -19,12 +19,12 @@
   },
   "peerDependencies": {
     "@spectrum-css/icon": "^3.0.0",
-    "@spectrum-css/vars": ">=9.0.0"
+    "@spectrum-css/tokens": "^8.1.0"
   },
   "devDependencies": {
-    "@spectrum-css/component-builder": "*",
+    "@spectrum-css/component-builder-simple": "*",
     "@spectrum-css/icon": "^3.0.38",
-    "@spectrum-css/vars": "^9.0.2",
+    "@spectrum-css/tokens": "^8.1.0",
```

Comment:
Should this get bumped to 9 now? The versions here might need a refresh.

Reply:
updated